### PR TITLE
#7 added all js bits 

### PR DESCRIPTION
**Pull request by ruarplum**

### CODE REVIEW

File: ruarplum/node.js [hunk 1]
```diff
@@ -6,17 +6,154 @@ console.log(x)
 // node node.js - prints to terminal
 
 
-
 // 2. Showing an if statement
 
+let password = "stephanie"
```
Comment (nmunro) [inline ruarplum/node.js:11]: Missing semicolon at end of line

File: ruarplum/node.js [hunk 2]
```diff
@@ -6,17 +6,154 @@ console.log(x)
...
+    console.log("password must be 8 characters or more");
+}
+else{
+    console.log("Your new password has been accepted");
+}
 
 
 // 3. Showing a switch case
 
+let topping = "pinapple"
```
Comment (nmunro) [inline ruarplum/node.js:23]: Another semicolon

File: ruarplum/node.js [hunk 3]
```diff
@@ -6,17 +6,154 @@ console.log(x)
...
+
+switch (topping){
+    case "Salami":
+    case "cheese":
+    case "chilli":
+    case "olives":
+    case "anchovies":
+        console.log("This is a key pizza topping")
```
Comment (nmunro) [inline ruarplum/node.js:31]: Semicolon missing from end of line

File: ruarplum/node.js [hunk 4]
```diff
@@ -6,17 +6,154 @@ console.log(x)
...
+
 // 4. Show an example of an array
 
-// 5. Show an example of a for loop
+let boatTypes = [
+    "jeanneau",
+    "hanse",
+    "swan",
+    "oyster"
+]
```
Comment (nmunro) [inline ruarplum/node.js:57]: Missing semicolon at end of line

File: ruarplum/node.js [hunk 5]
```diff
@@ -6,17 +6,154 @@ console.log(x)
...
+
+// // -----------------------------
+
+
+const square = (number) => number * number;
+const numbers = [1, 2, 3, 4, 5];
+const squares = numbers.map(square);
+const names = ["larry", "curly", "mo"];
+const shoutyNames = names.map(name => name.toUpperCase());
+const weirdlyCasedNames = names.map(name => {
```
Comment (nmunro) [inline ruarplum/node.js:113]: I recognise this code...

**Review by nmunro — APPROVED** [review]
Everything should run, but in addition to the comments I did make there's lines missing semicolons at the end of some lines. It wouldn't break anything so I'll approve it, but you might want to fix it up :)

**Review by jennifer-carey — APPROVED** [review]

**Pull request merged**